src/gui/userstatusselectormodel.cpp

```diff
@@ -406,23 +406,23 @@ QString UserStatusSelectorModel::timeDifferenceToString(int differenceSecs) cons
     } else if (differenceSecs < 60 * 60) {
         const auto minutesLeft = std::ceil(differenceSecs / 60.0);
         if (minutesLeft == 1) {
-            return tr("1 minute");
+            return tr("%1 minute", "", minutesLeft);
         } else {
-            return tr("%1 minutes").arg(minutesLeft);
+            return tr("%1 minutes", "", minutesLeft);
```

Comment:
you have to keep the `.arg(minutesLeft)` otherwise the placeholder will not be substituted